test/regress/cli/regress0/quantifiers/horn-ground-pre-post.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --sygus-inference
+; COMMAND-LINE: --sygus-inference=try
```

Comment:
I recommend doing to following for some subset (maybe just a couple) of the tests, so that the other option is tested as well:

```suggestion
; COMMAND-LINE: --sygus-inference=try
; COMMAND-LINE: --sygus-inference=on
```